common-api/src/main/resources/common_api.yaml

```diff
@@ -77,7 +77,7 @@ definitions:
 
   Gender:
     type: string
-    enum: &Gender [MALE, FEMALE, NON_BINARY, TRANSGENDER, INTERSEX, NONE, PREFER_NO_ANSWER]
+    enum: &Gender [MAN, WOMAN, NON_BINARY, TRANSGENDER, INTERSEX, NONE, PREFER_NO_ANSWER]
```

Comment:
If we receive `PREFER_NO_ANSWER`, should we disallow any other choices in the list?

Reply:
I agree prefer not to answer and Non of these describe me should have additional checks,  not just for GENDER but also for other field like SEX AT BIRTH, we do not have that logic as of now (even on UI) we _might_ introduce that later